src/DependencyInjection/Configuration.php

```diff
@@ -56,11 +56,11 @@ private function addConverterSection(ArrayNodeDefinition $rootNode): void
                                 ->arrayPrototype()
                                     ->beforeNormalization()
                                         ->ifNull()
-                                        ->then(fn () => ['source' => null, 'default' => null])
+                                        ->then(fn () => ['source' => null, 'default' => null, 'nullsafe' => null])
```

Comment:
I think we should use `false` here as well.
```suggestion
                                        ->then(fn () => ['source' => null, 'default' => null, 'nullsafe' => false])
```